frontend/pages/app.jsx

```diff
@@ -3,9 +3,11 @@ import PropTypes from 'prop-types';
 import Notifications from 'react-notification-system-redux';
 import { useSelector } from 'react-redux';
 import { Outlet, useLocation } from 'react-router-dom';
-
 import alertActions from '@actions/alertActions';
 import BuildStatusNotifier from '@util/buildStatusNotifier';
+import { createWrapper } from '@support/queryClient';
```

Comment:
I think we should mostly treat `support` as not part of the frontend code and replace this by inlining the `QueryClientWrapper` directly here (we can leave the support code for testing purposes)